# Introduction


## Why proteomics?

The human genome encodes ~20,000 different proteins, which are mole- cules that carry out the majority of biological functions. The identity and func- tion of a protein is defined by its unique sequence of 20 naturally-occurring amino acids connected through peptide bonds. Some common proteins are keratin, collagen, and hemoglobin. Most human diseases involve changes in the composition or functions of proteins in one form or another. Proteomics is a powerful technique that allows the identification and quantification of virtual- ly all the protein species in a tissue, i.e., its proteome, on a large-scale, through which researchers can gain important insights into the molecular makeup and physiology of a biological sample. At the Center, our investiga- tors use proteomics to compare the expression, interactions, and turnover of proteins in heart disease models to understand disease mechanisms.


## Experimental basics

EXPERIMENTAL WORKFLOW

experimental spectra
How do you identify and quantify proteins from biological samples? The tradi- tional way was to use antibodies that recognize specific proteins, but this ap- proach suffers from limited throughput. To identify proteins on a large scale, proteomics scientists use mass spectrometry. A mass spectrometer is like a balance for very small things: it can measure the accurate molecular weight of protein molecules.
Because mass spectrometers are not very good at analyzing entire proteins, protein samples are usually first digested into smaller fragments (peptides) prior to analysis. This is typically done with trypsin, a sequence-specific prote- ase that cleaves the protein sequence after every lysine (K) and arginine (R). The proteome is too complex for mass spectrometers to analyze all at the same time, so digested peptides are usually first resolved using liquid chro- matography, an analytical chemistry technique which separates the peptides into individual elution peaks over time, based on their chemical properties.
Once the peptides enter the mass spectrometer, they are ionized, and their mass is recorded as the precursor mass in a mass spectrum (MS1 spec- trum). Many peptides could have the same masses, however, so sequences are only deduced when the peptides are fragmented and the individual frag- ment masses recorded again in a tandem mass spectrum (MS2 spectrum). A mass spectrometry experiment could take hours to days, during which time spectra are continuously acquired. The ion signal intensity of a particular mass range over time is summarized in a chromatogram.

From the fragment masses on the MS2 spectrum, one can deduce the se- quence of the peptide that gave rise to the spectrum, and by extension the protein that gave rise to the peptide.

Although spectra could be interpreted manually to find peptide sequences, modern mass spectrometers generate hundreds of thousands of spectra per day, which is infeasible to analyze by hand. A database search method and algorithm is therefore commonly used to automatically identify peptides from spectra. A sequence database contains predicted protein sequencs from ge- nomic sequences, which can be used to generate theoretical spectra that contain predicted fragment ion masses. An algorithm is then used to evaluate the cross correlation between the theoretical and the experimental spectrum, and the best matched theoretical spectra are ranked.

An alternative to sequence database search is spectral library search. Em- pirically identified spectra from prior experiments are collected and deposited into a library, which can be used to directly identify peptides in subsequent experiments by matching new experimental spectra to the library spectra.
Mass spectrometers can also be used to quantify peptides and proteins. Two common methods are quantification based on spectral counts (number of spectra from which the peptide was identified) and ion intensity (the area-un- der-curve of the peptide ion signal). Additional methods including isotope la- beling and targeted MS can also be used for accurate measurements.